main.py

```diff
@@ -16,30 +16,37 @@ def main() -> QuerySet:
         ("Scarlett", "Johansson")
     ]
 
-    for genre in film_genres:       # creating Genre table (CREATE)
+    # creating records in Genre table (CREATE)
+    for genre in film_genres:
         Genre.objects.create(name=genre)
 
-    for first_name, last_name in actors:        # creating Actor table (CREATE)
+    # creating records in Actor table (CREATE)
+    for first_name, last_name in actors:
         Actor.objects.create(first_name=first_name, last_name=last_name)
 
     # read from Actors table (READ)
-    read_from_table = Actor.objects.filter(last_name="Smith").\
-        order_by("first_name")
+    retrieve = Actor.objects.filter(
```

Comment:
why did you decide that this name is better? Your variable name should describe what is stored in it. read or retrieve is an action but you are going to store data in this variable (people with the surname Smith).

Reply:
and the close bracket should be from the next line:
```
Actor.objects.filter(
    ...
).order_by("first_name")
```